bad_connector_woocommerce/views/product_views.xml

```diff
@@ -58,6 +58,7 @@
                                     <field name="price" />
                                     <field name="regular_price" />
                                     <field name="stock_management" />
+                                    <field name="downloadable_product" readonly="1" />
```

Comment:
remove from here and add at py level and write test case for downloadable product.